apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -153,9 +153,8 @@ class VirtualizedDetailView extends Component {
         {rowIndex > 1 && columnIndex > 0 && (
           <StudentProgressDetailCell
             studentId={section.students[studentStartIndex].id}
-            studentLevelProgress={studentLevelProgress}
             stageId={stageIdIndex}
-            scriptData={scriptData}
+            levelsWithStatus={getLevels(section.students[studentStartIndex].id, stageIdIndex)}
```

Comment:
nit: `section.students[studentStartIndex]` is used a few times here, maybe pull out a `student` variable.

Reply:
Because the row index changes (not always a student row), that variable would only be valid part of the time, so I felt it was nicer to just leave it as is.

Reply:
It could be cleaner to split those situations out into separate render functions, and not just for the sake of having a `student` variable.

Reply:
That's fair. I'll leave it for a follow up PR.